CellProfiler_pipelines/README.md

```diff
@@ -0,0 +1,107 @@
+The goal of having multiple pipelines using various methods will allow for benchmarking for the "best" method later on. 
+
+## Illumination Correction (IC) Pipeline
+
+In the [NF1_illum.cpproj](Pipelines/NF1_illum.cpproj) CP project, I perform illumination correction on each channel (DAPI, GFP, RFP) of the NF1 data. 
+There are three sets of `CorrectIlluminationCalculate` and `CorrectIlluminationApply` modules, where each set corrects a different channel and each contains different set of parameters as I have found using the same parameters for each channel does not always work.
+
+For more information regarding these modules, you can reference my [Illumination Correction prototyping README](https://github.com/WayScience/CellProfiler_Prototyping/tree/main/1.illumination_correction) within the [CellProfiler_Prototyping repo](https://github.com/WayScience/CellProfiler_Prototyping). 
+This README describes each of the parameters within these modules and provides further references.
+
+Since the NF1 pilot data is small, this IC pipeline saves all of the images to use in the next pipeline.
+For much larger datasets, it is standard to save the illumination correction function (which is much smaller and saves storage) and use that to perform IC in the next pipeline.
+
+## Analysis Pipeline
+
+In the [NF1_analysis.cpproj](Pipelines/NF1_analysis.cpproj) CP project, I perform segmentation and feature extraction.
+
+### Segmentation
+
+To perform segmentation in CP, the modules that were used were:
+
+- `IdentifyPrimaryObjects` Identify nuclei from the DAPI channel images. This creates a group for an object called `OrigNuclei`, which includes nuclei that are a part of cells that touch an edge of the image.
+
+- `IdentifySecondaryObjects` Identify whole cells using the nuclei from the previous module as a base from the RFP channel (stained for actin) images. 
+These whole cells make up the object group called `Cells`.
+Any cells that are touching at least one edge of the image will be removed along with its respective nucleus. 
+This creates a new object group call `Nuclei`.
+
+- `IdentifyTeritaryObjects` Identify cytoplasm by subtracting out the "smaller identfied objects" (`OrigNuclei`) from the "larger identified objects" (`Cells`). 
+`OrigNuclei` has to be used in this module over `Nuclei` due to an error downstream in the pipeline when using the latter object where it states "Cytoplasm is not in a 1:1 relationship with other objects".
+This means that the number of cytoplasm is not the same as the other objects, which would be wrong because there should be the same amount of objects (e.g the number of cells should match the number of cytoplasm and nuclei).
+
+Based on my understanding of CellProfiler, these modules identify the objects and create masks for each of them to use for feature extraction.
+
+### Feature Extraction
+
+By using the object masks, each module runs through each image from every channel to extract features from.
```

Comment:
I stumbled on this sentence. Can you rearrange? I think all the components are there, but maybe out of order.

How about something like:

"We apply each feature extraction module to each image (from every channel) using the aforementioned object masks.
This process measures features for cells, cytoplasm, and nuclei in each of our three channels."

Reply:
Fixed!